cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/AdapterLifecycleTests.java

```diff
@@ -103,6 +105,31 @@ public void testAdapterLifeCycle() throws Exception {
     JsonObject deployedApp = deployedApps.get(0);
     Assert.assertEquals(adapterType, deployedApp.get("id").getAsString());
 
+    List<ScheduleSpecification> schedules = getSchedules(namespaceId, adapterType, AdapterApp.AdapterWorkflow.NAME);
```

Comment:
This should check the adapter APIs and not querying the schedules directly. Let revamp the tests.